source/modules/MEI.header.xml

```diff
@@ -1454,7 +1454,7 @@
       </attDef>
     </attList>
     <remarks xml:lang="en">
```

Comment:
Please add a versionDate:
```suggestion
    <remarks xml:lang="en" versionDate="2024-08-13">
```

Reply:
What does this do? I thought the version will be `5.1`?

Reply:
Just for keeping track when descriptions/remarks were updated. Super annoying to go through commits trying to find out. (In use by TEI since 2005.)

Reply:
Uhm... is that the correct usage of `versionDate`? The description seems to indicate something about translation? 

https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-att.translatable.html

Reply:
Shouldn't it be "@change"? https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-att.global.change.html

Reply:
All languages carry it: https://github.com/TEIC/TEI/blob/dev/P5/Source/Specs/att.translatable.xml
We can discuss it in an ODD meeting.

Reply:
Yes, I know. But it seems to indicate that it is used to mark a version date for something that is being sent off to translation.

"specifies the date on which the source text was extracted and sent to the translator"

Which I don't think is what we want here?

Reply:
> We can discuss it in an ODD meeting.

I just looked, and I think the next ODD meeting I can make it to will be 29 November. Surely we can get this merged before then?

Reply:
It's just a suggestion, I already approved.

Reply:
I don't think having `@versionDate` is a very helpful practice. Maybe it was in 2005, but now that everything is fully versioned by git it does not make much sense any more. What would you expect to happen to `@versionDate` when the content of the remark is edited? To be updated or to remain the same? It is not satisfactory either way. Looking at the history is much more precise.

Reply:
I agree with @lpugin.